### PR TITLE
Add CollectionFS and $meteorCollectionFS Tutorial and Example

### PR DESCRIPTION
**Pull request by dotansimha**
- Add $meteorCollectionFS and CollectionFS tutorial
- Update CollectionFS example.

### CODE REVIEW

File: examples/collectionfs/.gitignore [hunk 1]
```diff
@@ -1,4 +0,0 @@
-.meteor/local
```
Comment (Urigo) [inline examples/collectionfs/.gitignore]: @dotansimha why?

File: examples/collectionfs/client/example.js [hunk 2]
```diff
@@ -1,24 +0,0 @@
-angular.module('example', ['angular-meteor']);
```
Comment (Urigo) [inline examples/collectionfs/client/example.js]: @dotansimha why did you remove the example and it's tests?

**Comment by dotansimha** [comment]
@Urigo 
Fixed! :)

**Comment by dotansimha** [comment]
@Urigo , please do not merge yet, missing tutorial for Edit & Crop image

**Comment by dotansimha** [comment]
@Urigo 
Done with the tutorial and added all examples! :)

**Comment by dotansimha** [comment]
@Urigo , Fixed tests. this caused by Travis that checkout the master for a reason, and then used the PR, so some of my changes was not available in examples/collectionfs/packages/angular-meteor symlink...
Fixed by using the real package of angular-meteor.
